### PR TITLE
[Vue] remove angularjs from CoreHome and CorePluginsAdmin

### PR DESCRIPTION
**Pull request by diosmosis**
### Description:

**This PR is based off every vue-remove-angularjs-<PluginName> PR and should be merged after they are all merged.**

Changes:
* Remove use of angularjs directives in twig templates.
* Remove use of ng-app/ng-cloak.
* Remove vue=>angularjs adapters in CoreHome/CorePluginsAdmin.
* Convert zen-mode.js to use Vue code.
* Add some extra output to vue:build + add a --bail option
* Create MobileSideNav vue component.
* Load menu translations client side if not in installation or updater (since plugins need to be available).

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by sgiehl** [comment]
I've pushed a couple of fixes, but there is still one js error remaining in frontend:
```
Segmentation.js?cb=9198:679 Uncaught TypeError: Cannot read properties of undefined (reading 'dispatchEvent')
    at closeForm (Segmentation.js?cb=9198:679:65)
    at HTMLBodyElement.<anonymous> (Segmentation.js?cb=9198:546:21)
    at HTMLBodyElement.dispatch (jquery.min.js?cb=9198:3:7537)
    at r.handle (jquery.min.js?cb=9198:3:5620)
```

**Comment by diosmosis** [comment]
@sgiehl I wasn't able to reproduce that specific error, but there were quite a few test a failures. I fixed those, hopefully it fixes whatever you were experiencing. If not, steps to reproduce would be helpful.

**Comment by sgiehl** [comment]
Seems my issue has been fixed. Will do a final review once the PRs in all submodule plugins have been reviewed and merged, so we can safely merge this one and afterwards the other PRs to remove angular.

**Comment by diosmosis** [comment]
submodules have been reviewed and merged

File: plugins/CoreHome/templates/_headerMessage.twig [hunk 1]
```diff
@@ -10,54 +10,26 @@
...
+    is-internet-enabled="{{ isInternetEnabled|json_encode }}"
+    update-check="{{ updateCheck|default|json_encode }}"
+    has-some-view-access="{{ hasSomeViewAccess|json_encode }}"
+    is-anonymous="{{ isUserIsAnonymous|json_encode }}"
+    contact-email="{{ contactEmail|json_encode }}"
+    piwik-version="{{ piwik_version|default(null)|json_encode }}"
+    style="display:inline;"
+></div>
 
+<span class="icon icon-reload hidden" title="{{ 'CoreHome_Refresh'|translate }}"></span>
```
Comment (sgiehl) [inline plugins/CoreHome/templates/_headerMessage.twig]: That icon had been reverted on 4.x-dev. Guess that needs to be removed here as well.

**Review by sgiehl — CHANGES_REQUESTED** [review]
Added a comment. Otherwise the code looks fine so far. Have not yet clicked through the UI much, will do that tomorrow...

**Comment by sgiehl** [comment]
@diosmosis would you mind checking why the javascript tests are failing?

**Review by sgiehl — APPROVED** [review]
Looks everything works now 👍

**Pull request merged**